java/core/src/java/org/apache/orc/impl/reader/StripePlanner.java

```diff
@@ -144,10 +165,23 @@ public StripePlanner parseStripe(StripeInformation stripe,
    * @return the buffers that were read
    */
   public BufferChunkList readData(OrcIndex index,
-                                boolean[] rowGroupInclude,
-                                boolean forceDirect) throws IOException {
+                                  boolean[] rowGroupInclude,
```

Comment:
Update doc here? how readLevel is changing the behaviour of the method?

Reply:
Will add